openedx/core/djangoapps/notifications/models.py

```diff
@@ -0,0 +1,64 @@
+"""
+Models for notifications
+"""
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
```

Comment:
Just a nit. Rest looks good


Disable pylint can be avoided using

from django.contrib.auth import get_user_model
User = get_user_model()


user = models.ForeignKey(User, related_name="notifications", on_delete=models.CASCADE)